kaolin/io/obj.py

```diff
@@ -156,15 +164,17 @@ def import_mesh(path, with_materials=False, with_normals=False, error_handler=No
         materials_order = None
 
     if with_normals:
-        vertex_normals = torch.FloatTensor([float(el) for sublist in vertex_normals
-                                            for el in sublist]).view(-1, 3)
+        vertex_normals = torch.FloatTensor(
+            [float(el) for sublist in vertex_normals
+             for el in sublist]).view(-1, 3)
         face_normals = torch.LongTensor(face_normals) - 1
     else:
         vertex_normals = None
         face_normals = None
 
-    return return_type(vertices, faces, uvs, face_uvs_idx, materials,
-                       materials_order, vertex_normals, face_normals)
+    return ObjMesh(vertices, faces, uvs, face_uvs_idx, materials,
```

Comment:
AFAIK nametuple shouldn't start with a capital letter.

Also I think we should keep it as `kaolin.io.obj.return_type`. it is pretty standard in pytorch, see https://pytorch.org/docs/stable/generated/torch.topk.html or https://pytorch.org/docs/stable/generated/torch.max.html ...

Will see later when we have a high level rep :)

Reply:
I followed the CamelCase convention since it's acts as a factory function: https://stackoverflow.com/questions/36371047/python-naming-convention-namedtuples

Also I think PyTorch did that since the return type of these operations are usually "anonymous" to the outside. However, I think the return type of `import_mesh` deserves its own meaningful name since it represents concrete data (i.e. a mesh) instead of just the result of some operation. As a user I would also want to import the name of this mesh "class" to annotate types or perform assertion, and it would make a bit more sense if I import `kaolin.io.obj.ObjMesh` instead of `kaolin.io.obj.return_type as obj_return_type`.

What do you think?

Reply:
I hear you, however we should stay consistent with https://github.com/NVIDIAGameWorks/kaolin/blob/master/kaolin/io/usd.py `mesh_return_type`. Also this is supposed to be anonymous, we don't want users to use this nametuple to create their own (and it's not necessary to import the type to run assertion), this is the output of the obj loader, not a standard mesh representation (which will be implemented later on, if we start to have this nametuple as a standard rep then it will be confused with the higher rep later), I think it's better to keep it as `return_type` or `obj_return_type`. There should be no multiple return_type from obj loader.